tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -330,9 +330,9 @@ describe('E2E Testing for taqueria typechecker and simulator tasks of the tezos-
 	afterAll(async () => {
 		try {
 			await exec(`taq stop sandbox ${dockerName}`, { cwd: `./${taqueriaProjectPath}` });
+			await fsPromises.rm(taqueriaProjectPath, { recursive: true });
```

Comment:
There is an issue with moving it into try block. It won't be triggered if there is an exception.